### PR TITLE
Adds in Google Analytics Events attributes and bumps version to 24.2.5

### PR DESCRIPTION
**Pull request by DilwoarH**

### CODE REVIEW

File: toolkit/templates/documents.html [hunk 1]
```diff
@@ -1,16 +1,16 @@
 <ul class='document-list'>
   {% for item in items %}
     <li class='document-list-item'>
-      {% if item.file_type %}
-        <a href="{{ item.link }}" class='document-link-with-icon' {% if item.download %}download {% endif %}>
-          <span class='document-icon'>{{ item.file_type }}<span> document:</span></span>
+        <a href="{{ item.link }}" class='document-link{% if item.file_type %}-with-icon{% endif %}'
```
Comment (galund) [inline toolkit/templates/documents.html]: picky-picky but this is now indented too far :-)

File: toolkit/templates/documents.html [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-        <a href="{{ item.link }}" class='document-link-with-icon' {% if item.download %}download {% endif %}>
-          <span class='document-icon'>{{ item.file_type }}<span> document:</span></span>
+        <a href="{{ item.link }}" class='document-link{% if item.file_type %}-with-icon{% endif %}' 
+          {% if item.download %}download {% endif %}
+          {% if item.analytics %}data-analytics="{{ item.analytics }}" {% endif %}
+          {% if item.analytics_category %}data-analytics-category="{{ item.analytics_category }}" {% endif %}
+          {% if item.analytics_action %}data-analytics-action="{{ item.analytics_action }}" {% endif %}
+          {% if item.analytics_label %}data-analytics-label="{{ item.analytics_label }}" {% endif %}
+        >
+          {% if item.file_type %}<span class='document-icon'>{{ item.file_type }}<span> document:</span></span>{% endif %}
```
Comment (galund) [inline toolkit/templates/documents.html]: Oh, I'd still like to get rid of the extra span, if we can

Comment (DilwoarH) [reply toolkit/templates/documents.html]: screen shot 2017-10-16 at 10 05 14

It's needed by css, if we get rid of it the icons don't work.

**Comment by DilwoarH** [comment]
indentation has been fixed. Merging.

**Pull request merged**